new/package.json

```diff
@@ -2,64 +2,26 @@
   "name": "new",
   "version": "0.1.0",
   "private": true,
-  "devDependencies": {
-    "@amcharts/amcharts5": "^5.3.7",
-    "parcel": "^2.6.2"
-  },
   "dependencies": {
-    "@cubejs-client/core": "^0.31.0",
-    "@devexpress/dx-react-chart-material-ui": "^4.0.3",
-    "@emotion/styled": "^11.10.6",
-    "@mui/icons-material": "^5.11.11",
-    "@mui/material": "^5.11.10",
-    "@mui/x-data-grid": "^5.17.25",
-    "@syncfusion/ej2-react-charts": "^20.4.53",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "@types/jest": "^27.5.2",
     "@types/node": "^16.18.3",
     "@types/react": "^18.0.25",
     "@types/react-dom": "^18.0.9",
-    "amcharts": "^3.14.5",
-    "chart.js": "^4.2.1",
-    "i18next": "^22.4.10",
     "react": "^18.2.0",
-    "react-chartjs-2": "^5.2.0",
     "react-dom": "^18.2.0",
-    "react-donut-chart": "^1.3.3",
-    "react-i18next": "^12.2.0",
     "react-scripts": "5.0.1",
     "typescript": "^4.9.3",
-    "utility-types": "^3.10.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "bash -c \"tsc && react-scripts test\"",
+    "test": "react-scripts test",
     "eject": "react-scripts eject"
   },
-  "prettier": {
```

Comment:
shouldn't we keep the prettier config?

Reply:
Perhaps. Some stuff seem to have changed without need -.-